apps/sudoku-backend/src/app/sudoku/models/sudoku-field.dto.ts

```diff
@@ -2,10 +2,14 @@ import { Min, Max, IsNumber } from 'class-validator';
 import { ApiProperty } from '@nestjs/swagger';
 import { Exclude, Expose } from 'class-transformer';
 import { ISudokuFieldDto } from '../../../../../../libs/models/sudoku-field.dto';
```

Comment:
Glaub es würd sinn machen an alias path zu dem libs folder zum machen. 
Damit ma schreiben könnt `@libs/models/sudoku-field.dto`
Würd wesentlich besser ausschauen ois des `../../../../../../libs`

Is aber natürlich ned für den pull request, aber halt so allgemeine idee